test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -78,11 +78,27 @@ public EndToEndTestsBase(TTestFixture fixture)
             {
                 { "input",  "Hello" }
             };
-            
-            //Only verifying the call succeeds. It is not possible to verify
-            //actual push notificaiton is delivered as they are sent only to 
-            //client applications that registered with NotificationHubs
-            await Fixture.Host.CallAsync("NotificationHubOut", arguments);
+
+            try
+            {
+                //Only verifying the call succeeds. It is not possible to verify
+                //actual push notificaiton is delivered as they are sent only to 
+                //client applications that registered with NotificationHubs
+                await Fixture.Host.CallAsync("NotificationHubOut", arguments);
```

Comment:
You need to change this to not be a hardcoded "NotificationHubOut" string. That means that all of your tests actually run the same function. Instead, let this take a `functionName` parameter and have each of the tests pass that in. 

I ran into this today when testing my multi-config settings. The NotificationHubOutNotification test kept passing when it shouldn't have -- it didn't have any HubName. Turns out it was actually running NotificationHubOut b/c this string was hardcoded.